agent/metrics/metrics.go

```diff
@@ -0,0 +1,214 @@
+/*
+Copyright 2019 Google Inc. All rights reserved.
```

Comment:
Shouldn't this say "2021" instead of "2019"?

Reply:
Done